systemd/systemd.go

```diff
@@ -301,6 +302,22 @@ func (s *systemd) Status(serviceNames ...string) ([]*ServiceStatus, error) {
 	return sts, nil
 }
 
+// IsEnabled checkes whether the given service is enabled
+func (s *systemd) IsEnabled(serviceName string) (bool, error) {
+	_, err := systemctlCmd("--root", s.rootDir, "is-enabled", serviceName)
+	if err == nil {
+		return true, nil
+	}
+	// note that "systemctl is-enabled <name>" returns exit code 1 for disabled services
```

Comment:
IMO it's enough to describe why we're doing things, what is done is seen in the code. I'd rephrase the comment to something like:

```
// "systemctl is-enabled <name>" prints `disabled\n` to stderr and returns exit code 1 
// for disabled services
```

Reply:
This is done.